internal/cirrusenv/cirrusenv.go

```diff
@@ -0,0 +1,71 @@
+package cirrusenv
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/google/uuid"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type CirrusEnv struct {
+	file *os.File
+}
+
+func New() (*CirrusEnv, error) {
+	cirrusEnvFile, err := os.Create(filepath.Join(os.TempDir(), "cirrus-env-"+uuid.New().String()))
```

Comment:
Can a task id be used here?

Reply:
`CIRRUS_ENV` file is unique for each instruction, this makes sure that if the current instruction (1) deletes this file (2) or corrupts it accidentally, the other instruction will be able to work just fine.

Would including a task ID before UUID help?

Reply:
Ok, let's at least append task id in case of future debugging.

Reply:
Added in https://github.com/cirruslabs/cirrus-ci-agent/pull/144/commits/80a9c5e6cdbbc741dc20bcdc1290c77623a8e756.